app/helpers/restaurant_helper.rb

```diff
@@ -0,0 +1,2 @@
+module RestaurantHelper
+end
```

Comment:
Why are these here? Please do not use generators or scaffolds. We dont want these empty bloat files